sdk/core/azure-core-amqp/samples/local_server_sample/local_server_sample.cpp

```diff
@@ -163,9 +166,9 @@ class SampleEvents : public ConnectionEvents,
       LinkEndpoint& newLink,
       std::string const& name,
       Azure::Core::Amqp::_internal::SessionRole,
-      Azure::Core::Amqp::Models::AmqpValue source,
-      Azure::Core::Amqp::Models::AmqpValue target,
-      Azure::Core::Amqp::Models::AmqpValue) override
+      Azure::Core::Amqp::Models::AmqpValue const&source,
+      Azure::Core::Amqp::Models::AmqpValue const&target,
+      Azure::Core::Amqp::Models::AmqpValue const&) override
```

Comment:
You have a few more instances of this, comenting only on the first one.
Question 2: why does not the last AmqpValue parameter has a name?
```suggestion
      Azure::Core::Amqp::Models::AmqpValue const& source,
      Azure::Core::Amqp::Models::AmqpValue const& target,
      Azure::Core::Amqp::Models::AmqpValue const&) override
```

Reply:
I have a convention of leaving parameters which are not referenced in the function without naming them. It generally makes the code simpler (saves from having to write `(void)unreferencedParameter`).

If you ever see a parameter which is not named, then it is safe to assume that the parameter is not referenced in the code.